types/types.go

```diff
@@ -23,67 +24,177 @@ import (
 	"github.com/prometheus/common/model"
 )
 
+type State uint8
+
+const (
+	Unprocessed State = iota
```

Comment:
Always prefix enums with the original type.

The type should be `AlertState` to know what it's even about. Thereby the values would be `AlertStateUnprocessed` etc.